docs/src/main/asciidoc/websockets-next-reference.adoc

```diff
@@ -661,6 +661,47 @@ class MyBean {
 ----
 <1> An asynchronous observer method is executed using the default blocking executor service.
 
+[[server-tenants]]
+==== Tenants
```

Comment:
The title should be something like "Hibernate multitenancy"? Also it would not hurt to add the link to the Hibernate docs: https://quarkus.io/guides/hibernate-orm#multitenancy

Reply:
And the section should IMO go after https://quarkus.io/guides/websockets-next-reference#tls at the same level (e.g. using `===`).

Reply:
Agreed.